### PR TITLE
Add basic stack wizard backend

### PR DESCRIPTION
**Pull request by masonforest**

### CODE REVIEW

File: db/migrate/20150512194822_create_wizard_answers.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateWizardAnswers < ActiveRecord::Migration
+  def change
+    create_table :wizard_answers do |t|
+      t.references :wizard_question, index: true, foreign_key: true
+      t.string :answer
+      t.string :positive_tags
+      t.string :negative_tags
```
Comment (calebthompson) [inline db/migrate/20150512194822_create_wizard_answers.rb]: These names aren't super meaningful. Can we call them `tags_to_add` and `tags_to_remove` instead?

File: spec/requests/wizard_questions_spec.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        next_question_id: next_question.id,
+        wizard_answers:
+          [
+            {
+              answer: 'PHP',
+              positive_tags: 'PHP, Linux',
+              negative_tags: 'Windows, Java, Ruby, dotNet'
+            }.stringify_keys
+          ]
+      }.stringify_keys
```
Comment (calebthompson) [inline spec/requests/wizard_questions_spec.rb]: http://apidock.com/rails/v4.0.2/Hash/deep_stringify_keys

**Comment by calebthompson** [comment]
A couple minor things, then :peach:

**Comment by chriskacerguis** [comment]
:+1:

**Pull request merged**